src/hooks/queryHooks.ts

```diff
@@ -7,21 +7,37 @@ import {
   ShardInfo,
   TableInfo,
   TableListEntry,
-  User,
+  UserInfo,
 } from 'types/cratedb';
 import useExecuteSql from 'hooks/useExecuteSql';
 import {
   clusterInfoQuery,
   getPartitionedTablesQuery,
   nodesQuery,
   shardsQuery,
+  usersQuery,
 } from 'constants/queries';
+import { QueryResultSuccess } from 'types/query';
 
-export const useGetUsersQuery = () => {
+export const postFetchUsers = (data: QueryResultSuccess): UserInfo[] => {
```

Comment:
@joncombe this will probably be used in your PR as well. I needed this in the tests, and since you are working on it and I know you will create separate files I have simply put it inside this file. I guess the `postFetch` that you have introduced is useful even before you merge it 😁